### ISSUE TITLE
[BUG] When installing chia via pip, it does not work

### ISSUE THREAD

**Issue by ast0815** [Chia-Network/chia-blockchain#1153]
**Describe the bug**
When installing chia via pip, it does not work.

**To Reproduce**

```
$ virtualenv -p python3 env
$ . env/bin/activate
$ pip install chia-blockchain==1.0rc4
$ chia help
Traceback (most recent call last):
  File "/home/koch/Projekte/chia/env/bin/chia", line 5, in <module>
    from src.cmds.chia import main
  File "/home/koch/Projekte/chia/env/lib/python3.7/site-packages/src/cmds/chia.py", line 7, in <module>
    from src.daemon.server import async_run_daemon
  File "/home/koch/Projekte/chia/env/lib/python3.7/site-packages/src/daemon/server.py", line 16, in <module>
    from src.cmds.init import chia_init
  File "/home/koch/Projekte/chia/env/lib/python3.7/site-packages/src/cmds/init.py", line 10, in <module>
    from src.util.keychain import Keychain
  File "/home/koch/Projekte/chia/env/lib/python3.7/site-packages/src/util/keychain.py", line 14, in <module>
    from src.util.hash import std_hash
  File "/home/koch/Projekte/chia/env/lib/python3.7/site-packages/src/util/hash.py", line 3, in <module>
    from src.types.blockchain_format.sized_bytes import bytes32
ModuleNotFoundError: No module named 'src.types.blockchain_format'
```

**Expected behavior**
I would expect the CLI to work.

**Desktop (please complete the following information):**

- OS: Debian on WSL2

**Comment by ast0815** [comment]
Seems like chia installs a package "src" instead of one that is named "chia" or "chia_blockchain".

**Comment by hoffmang9** [comment]
the workaround is to use `git clone https://github.com/Chia-Network/chia-blockchain.git` for now.

**Comment by hoffmang9** [comment]
This should be fixed in 1.0.1
